addons/share-through-mystuff/userscript.js

```diff
@@ -0,0 +1,28 @@
+export default async function ({ addon, global }) {
+  const shareFunction = document.createElement("a");
+  shareFunction.classList.add("media-share");
+  shareFunction.dataset.control = "share";
+  shareFunction.href = "#";
+
+  const shareButton = document.createElement("a");
+  function shareConfirmation(event) {
+    let confirmation = confirm("Would you like to share this project?");
```

Comment:
I'll soon change this to use `addon.tab.confirm()` instead

Reply:
Steve0Greatness has updated the code and it now uses `addon.tab.confirm()`.